buffer/src/main/java/io/netty/buffer/PoolSubpage.java

```diff
@@ -115,7 +115,10 @@ boolean free(PoolSubpage<T> head, int bitmapIdx) {
 
         if (numAvail ++ == 0) {
             addToPool(head);
-            return true;
+            // when maxNumElems == 1, cannot return directly, also need to go through the following judgment
```

Comment:
Please reword... This comment doesn't really explain what this is all about.

Reply:
If `maxNumeLems == 1`, the maximum `numAvail` is also 1, and each free operation of the `PoolSubpage` will return in the branch of `numAvail ++ == 0`, which means the following code is unreachable

how about this?

Reply:
If `maxNumElems == 1`, the maximum `numAvail` is also 1. The free operation of this type of `PoolSubPage` will end here, which means that subsequent code is not reachable, these `PoolSubPages` will not really be recycled, so the `maxNumElems > 1` judgment needs to be added

Reply:
sounds good... Please update the comment to something like this^^^

Reply:
alright.